airbyte-integrations/connectors/source-stripe/unit_tests/integration/test_persons.py

```diff
@@ -0,0 +1,632 @@
+            _create_response().with_record(record=_create_record("events")).with_record(record=_create_record("events")).build(),
+        )
+
+        source = SourceStripe(config=_CONFIG, catalog=_create_catalog())
+        actual_messages = read(source, config=_CONFIG, catalog=_create_catalog())
+
+        assert emits_successful_sync_status_messages(actual_messages.get_stream_statuses(_STREAM_NAME))
+        assert len(actual_messages.records) == 4
+
+    @HttpMocker()
+    def test_accounts_400_error(self, http_mocker: HttpMocker):
+        http_mocker.get(
+            _create_request("accounts").with_limit(100).build(),
+            a_response_with_status(400),
+        )
+
+        source = SourceStripe(config=_CONFIG, catalog=_create_catalog())
+        actual_messages = read(source, config=_CONFIG, catalog=_create_catalog())
+        error_log_messages = [message for message in actual_messages.logs if message.log.level == Level.ERROR]
+
+        # For Stripe, streams that get back a 400 or 403 response code are skipped over silently without throwing an error as part of
+        # this connector's availability strategy
+        assert len(actual_messages.get_stream_statuses(_STREAM_NAME)) == 0
+        assert len(error_log_messages) > 0
+
+    @HttpMocker()
+    def test_persons_400_error(self, http_mocker: HttpMocker):
+        http_mocker.get(
+            _create_request("accounts").with_limit(100).build(),
+            _create_response().with_record(record=_create_record("accounts")).build(),
+        )
+
+        # Persons stream first page request
+        http_mocker.get(
+            _create_request("persons").with_limit(100).build(),
+            a_response_with_status(400),
+        )
+
+        source = SourceStripe(config=_CONFIG, catalog=_create_catalog())
+        actual_messages = read(source, config=_CONFIG, catalog=_create_catalog())
+        error_log_messages = [message for message in actual_messages.logs if message.log.level == Level.ERROR]
+
+        # For Stripe, streams that get back a 400 or 403 response code are skipped over silently without throwing an error as part of
+        # this connector's availability strategy. They are however reported in the log messages
+        assert len(actual_messages.get_stream_statuses(_STREAM_NAME)) == 0
+        assert len(error_log_messages) > 0
+
+    @HttpMocker()
+    def test_accounts_401_error(self, http_mocker: HttpMocker):
+        http_mocker.get(
+            _create_request("accounts").with_limit(100).build(),
+            a_response_with_status(401),
+        )
+
+        source = SourceStripe(config=_CONFIG, catalog=_create_catalog())
+        actual_messages = read(source, config=_CONFIG, catalog=_create_catalog(), expecting_exception=True)
+
+        assert actual_messages.errors[-1].trace.error.failure_type == FailureType.system_error
+
+    @HttpMocker()
+    def test_accounts_401_error(self, http_mocker: HttpMocker):
```

Comment:
It seems this test name is a duplicate from https://github.com/airbytehq/airbyte/pull/33955/files#diff-99758944345d58421790205c6abb00a0bbaffe9f6dc024f3bd9e1384925eabd2R213 Should it be `test_persons_401_error`?